### PR TITLE
[8.0][ADD] l10_cn_fapiao

### PR DESCRIPTION
**Pull request by franksongfeng**

### CODE REVIEW

File: l10n_cn_fapiao/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline l10n_cn_fapiao/__init__.py:1]: @franksongfeng   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: l10n_cn_fapiao/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import fapiao
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (elicoidal) [inline l10n_cn_fapiao/__init__.py]: remove this comments

File: l10n_cn_fapiao/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
```
Comment (elicoidal) [inline l10n_cn_fapiao/__init__.py]: Copyright is 2014-2017

File: l10n_cn_fapiao/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Chinese Fapiao Management',
+    'version': '8.0.0.0.1',
+    'category': 'Accounting',
+    'sequence': 19,
```
Comment (elicoidal) [inline l10n_cn_fapiao/__openerp__.py]: remove

File: l10n_cn_fapiao/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Chinese Fapiao Management',
+    'version': '8.0.0.0.1',
```
Comment (elicoidal) [inline l10n_cn_fapiao/__openerp__.py]: 8.0.1.0.0

File: l10n_cn_fapiao/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    'category': 'Accounting',
+    'sequence': 19,
+    'summary': 'Chinese Fapiao Management and link with OpenERP Invoice',
+    'description': """
```
Comment (elicoidal) [inline l10n_cn_fapiao/__openerp__.py]: Description should be moved to the README.rst file

File: l10n_cn_fapiao/fapiao.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    _name = "fapiao"
+    _inherit = 'mail.thread'
+    _order = "fapiao_date desc"
+
+    fapiao_type = fields.Selection(
+        [('customer', 'Customer'), ('supplier', 'Supplier'),
+         ('customer_credit_note', 'Customer Credit note')],
+        'Fapiao Type', required=True, default='customer')
+    tax_type = fields.Selection(
+        [('13%', '13%'), ('17%', '17%'),
```
Comment (elicoidal) [inline l10n_cn_fapiao/fapiao.py:17]: Not sure this is all possible values.
Maybe we need a comodel

File: l10n_cn_fapiao/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+* Step 1: A fapiao is received or emitted
+* Step 2: The document can be scanned as image
+* Step 3: A new fapiao is created in OpenERP
+* Step 4: The scanned document is uploaded to the newly created object as attachment.
+* Step 5: Fapiao allocation to invoice: Fapiao and OpenERP invoices can be linked to each other
+    """,
+    'author': 'Elico Corp, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'https://www.elico-corp.com',
+    'images': [],
```
Comment (elicoidal) [inline l10n_cn_fapiao/__openerp__.py]: add the image

File: l10n_cn_fapiao/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    'depends': ['account'],
+    'data': [
+        'fapiao_view.xml',
+    ],
+    'test': [
```
Comment (elicoidal) [inline l10n_cn_fapiao/__openerp__.py]: remove

**Review by elicoidal — CHANGES_REQUESTED** [review]
One important question (to be checked with local CPA)
I miss the translation file
Some modifications
Please add Unit tests

File: l10n_cn_fapiao/__init__.py (hunk 1 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__init__.py:1]: yes, follow these specs

File: l10n_cn_fapiao/__init__.py (hunk 2 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__init__.py]: fixed

File: l10n_cn_fapiao/__init__.py (hunk 3 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__init__.py]: fixed

File: l10n_cn_fapiao/__openerp__.py (hunk 5 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__openerp__.py]: fixed.

File: l10n_cn_fapiao/__openerp__.py (hunk 6 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__openerp__.py]: fixed.

File: l10n_cn_fapiao/__openerp__.py (hunk 8 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__openerp__.py]: fixed

File: l10n_cn_fapiao/__openerp__.py (hunk 9 above)
Comment (franksongfeng) [reply l10n_cn_fapiao/__openerp__.py]: fixed

File: l10n_cn_fapiao/fapiao.py (hunk 7 above)
Comment (elicoidal) [reply l10n_cn_fapiao/fapiao.py:17]: @franksongfeng can you chec with Connie whether it should be better to have comodel?

**Comment by elicoidal** [comment]
@franksongfeng we need a comodel for the tax selection

**Comment by elicoidal** [comment]
superseded by #41

**Pull request closed without merging**